dotcom-rendering/src/components/ArticleHeadline.stories.tsx

```diff
@@ -715,69 +721,9 @@ MatchReport.decorators = [
 	]),
 ];
 
-export const SpecialReport: StoryObj = ({ format }: StoryArgs) => {
-	return (
-		<Section fullWidth={true}>
-			<Flex>
-				<LeftColumn borderType="full">
-					<></>
-				</LeftColumn>
-				<ArticleContainer format={format}>
-					<ArticleHeadline
-						headlineString="This is the headline you see when pillar is SpecialReport"
-						format={format}
-						tags={[]}
-						webPublicationDateDeprecated=""
-					/>
-				</ArticleContainer>
-			</Flex>
-		</Section>
-	);
-};
-SpecialReport.storyName = 'SpecialReport';
-SpecialReport.decorators = [
-	lightDecorator([
-		{
-			display: ArticleDisplay.Standard,
-			design: ArticleDesign.Standard,
-			theme: ArticleSpecial.SpecialReport,
-		},
-	]),
-];
```

Comment:
Is this gone because it’s covered in another story?

Reply:
Yes, I added a [story](https://63e251470cfbe61776b0ef19-mmokcxjloz.chromatic.com/?path=/story/components-articleheadline--article-story) with all [theme variations](https://github.com/guardian/dotcom-rendering/pull/9759/files#diff-bd7128af36961b6d1504cdeb4a583d6ffc9c57622091307bdf4859b0209a4a9aR70)